client/src/components/audit-row/resolver.ts

```diff
@@ -0,0 +1,103 @@
+import { Audit } from '../../types';
+
+type Data = { [key: string]: never };
+
+interface IResolver {
+  (params: Data, query: Data, payload: Data): string;
```

Comment:
I would return `React.ReactNode` instead of string just in case we want more capabilities